assertions/assertions_test.go

```diff
@@ -1346,6 +1374,13 @@ func TestShouldHappenBetween(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
```

Comment:
This test is failing:

assertions_test.go:1388: ShouldHappenBetween() error = expected '2023-09-07 09:06:34.28230254 +0000 UTC m=+0.000668682' to be between '2023-09-07 01:00:00 +0000 UTC' and '2023-09-07 01:00:00 +0000 UTC' , wantErr false

Reply:
can you check again?

Reply:
@rhzs it's ok, thank you. LGTM